index.html

```diff
@@ -1847,9 +1847,9 @@ <h2>Verification Relationships</h2>
       </p>
       <p>
 The <a>verification relationship</a> between the <a>DID subject</a> and the
-<a>verification method</a> MUST be explicit in the <a>DID document</a>.
+<a>verification method</a> is explicit in the <a>DID document</a>.
 <a>Verification methods</a> that are not associated with a particular
-<a>verification relationship</a> MUST NOT be used for that <a>verification
+<a>verification relationship</a> cannot be used for that <a>verification
 relationship</a>. For example, a <a>verification method</a> in the value of
 the <code><a>authentication</a></code> property cannot be used to engage in
```

Comment:
Just noting that I tried to remove this normative language in an earlier PR based on a comment by @kdenhartog, and @msporny [objected on the basis that it is testable](https://github.com/w3c/did-core/pull/429#pullrequestreview-501574027). Given this is a conflict between present manu and past manu, I'm going to suggest manu can make the call :)

Reply:
*\*lol\** ... Dammit, past manu... 'causing trouble.

Past manu was wrong, @rhiaro and @kdenhartog are right.

The normative-ness of the statement is being removed because it is a test intended for a conformance class of software that we don't define in DID Core. In other words, it has very little to do with production and consumption conformance classes and instead has more to do with software that implements authentication/authorization protocols.

Present manu is displeased that past manu failed to realize that. :P